kube-client/Cargo.toml

```diff
@@ -16,7 +16,7 @@ rust-version = "1.60.0"
 edition = "2021"
 
 [features]
-default = ["client", "openssl-tls"]
```

Comment:
I believe this change was committed accidentally.

Reply:
reverted it